axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -263,29 +324,29 @@
           "salt": "Operators devnet-amplifier"
         },
         "InterchainTokenService": {
-          "salt": "ITS v2.0.0rc3 devnet-amplifier",
+          "salt": "ITS v2.0.1 devnet-amplifier2",
           "deployer": "0xba76c6980428A0b10CFC5d8ccb61949677A61233",
-          "proxySalt": "ITS v2.0.0 devnet-amplifier",
-          "tokenManagerDeployer": "0x4cE9EC3ECB34aD0284f77BF6AafdBef9d136bd24",
-          "interchainToken": "0x34c8C8d2E212E3D5eBECAdf4769388B2e36C4561",
-          "interchainTokenDeployer": "0x8Be3C8bbFAFeE70AEAC14406a5F5C6dCD4D4fCb0",
-          "tokenManager": "0x64291364740eb10aeC672f06eCD7c7a91ea5A752",
-          "tokenHandler": "0x445C50CC6085ff2d35364bcc3CDce8b3FE08b27d",
-          "gatewayCaller": "0xFE0144B9C5B487Ec0a6Db38F7f100E99125117b3",
-          "implementation": "0x110D33345f031b9b630366c8399838bE445fF22d",
+          "proxySalt": "ITS v1.0.0 devnet-amplifier2",
+          "tokenManagerDeployer": "0x4Bf2315e29CacA44227E81c7484B4AF1dc7F4ED9",
+          "interchainToken": "0x5321CB9412D5F53d95D2B5a07861Dd45CBca2671",
+          "interchainTokenDeployer": "0x07DD1cE692d6D1f9DD02A81277DC06C79FFecde0",
+          "tokenManager": "0x4760fB5613c2818C85346805266B356000f256cC",
+          "tokenHandler": "0x836bF27091C254b5c8Da85408c95ED1E4A763a43",
+          "gatewayCaller": "0x57B8D2082E0C9a38d928F40C0B7E79e4BF8D703e",
+          "implementation": "0x733Ee16Fe75988b09119D698584E1d77a9FB4453",
           "predeployCodehash": "0x08a4a556c4db879b4f24104d13a8baf86915d58b12c81b382dfea2a82d2856cf",
-          "address": "0x144c3d7A5f5198EF3B46A8258b35E903cf197A66"
+          "address": "0x2269B93c8D8D4AfcE9786d2940F5Fcd4386Db7ff"
         },
         "InterchainTokenFactory": {
           "deployer": "0xba76c6980428A0b10CFC5d8ccb61949677A61233",
-          "salt": "ITS Factory v2.0.0 devnet-amplifier",
-          "implementation": "0x429E1bA4EF86B684741D04a97f18B25E78b8A8f1",
-          "address": "0x6Ae8C8498d5FDA930e6ABeB0E15E5A00471702a7"
+          "salt": "ITS Factory v1.0.0 devnet-amplifier2",
+          "implementation": "0x38a1DF03Ed0308fA545D60a458904Ad9416c8c65",
+          "address": "0xdB7d6A5B8d37a4f34BC1e7ce0d0B8a9DDA124871"
         }
       }
     },
     "eth-sepolia": {
-      "name": "Ethereum Sepolia",
+      "name": "Eth Sepolia",
```

Comment:
```suggestion
      "name": "Ethereum Sepolia",
```
more readable

Reply:
I changed this because something failed when trying to verify contract with name `Ethereum Sepolia`

Reply:
i see, need to make it use axelarId perhaps